Cargo.toml

```diff
@@ -1,7 +1,21 @@
 [package]
-name = "balena-configuration-dsl"
+name = "balena_configuration_dsl"
```

Comment:
Use `-`, not `_`. It's perfectly okay, looks nicer. The only place where one has to use `_` is in the `use` statement (or full path) in the code.